src/main/scala/nl.knaw.dans.easy.multideposit/parser/ParserValidation.scala

```diff
@@ -91,4 +94,18 @@ trait ParserValidation extends DebugEnhancedLogging {
       case accs => ParseError(deposit.row, s"Multiple accessibility levels found for A/V files: ${ accs.mkString("{", ", ", "}") }").toInvalid
     }
   }
+
+  def checkAllAVFileNamesWithoutSpaces(deposit: Deposit): Validated[Unit] = {
+    deposit.files.collect { case fmd: AVFileMetadata => fmd.filepath.name }
+      .toList
+      .traverse(name => noSpaces(deposit.row, name))
+      .map(_ => ())
+  }
+
+  private def noSpaces(row: Int, s: String): Validated[String] = {
+    if (!s.contains(' '))
+      s.toValidated
+    else
+      ParseError(row, s"A/V filename '$s' contains spaces").toInvalid
```

Comment:
usually it's the negation of the check that goes in the `else` branch

```suggestion
    if (s contains ' ')
      ParseError(row, s"A/V filename '$s' contains spaces").toInvalid
    else
      s.toValidated
```

Reply:
done